soh/src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -209,12 +226,57 @@ void func_809DF730(EnCow* this, GlobalContext* globalCtx) {
     }
 }
 
+bool EnCow_IsSecondCowInTower(EnCow* this, GlobalContext* globalCtx) {
+    return 
+        gSaveContext.n64ddFlag && 
```

Comment:
does it make sense to have the rando check in here? the cow is still the second cow in the tower even if we're not randomized, so it seems to me it would make more sense to keep those checks separate

so instead of having
```c
if (EnCow_IsSecondCowInTower(thisx, globalCtx)) {
```
we'd have
```c
if (gSaveContext.n64ddFlag && EnCow_IsSecondCowInTower(thisx, globalCtx)) {
```